src/test/java/com/softdevsix/api/domain/report/ReportTest.java

```diff
@@ -0,0 +1,51 @@
+package com.softdevsix.api.domain.report;
+
+import com.softdevsix.api.exceptions.ReportReaderException;
+import com.softdevsix.api.services.report.IReportReader;
+import com.softdevsix.api.services.report.JsonReportReader;
+import org.junit.jupiter.api.Test;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+class ReportTest {
+    public static String readFile(String path) {
```

Comment:
Don't you think this method should go into `JsonReportReader` class?

Reply:
Done